src/actions/urlCheckLoserboard.action.ts

```diff
@@ -0,0 +1,10 @@
+import { Message } from "discord.js";
+import { ServerValue } from "firebase-admin/database";
+import { db, CLIENT, leaderboards } from "../raiha";
+
+export async function urlCheckLoserboard(guildId: string, msgAuthor: Message.author.id<true>) {
```

Comment:
See the other comment where this function is called.
In that case, the parameter will be of type `Message<true>`, and you can pull out the ids here.